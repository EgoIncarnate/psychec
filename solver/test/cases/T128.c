void writeline (struct linebuffer const *line) {}
void compare_files()
{
    struct linebuffer *thisline[2];
    writeline (thisline[1]);
}
