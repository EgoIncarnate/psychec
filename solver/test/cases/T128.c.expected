
typedef  struct linebuffer {} const linebuffer ;
